src/status.h

```diff
@@ -17,12 +17,12 @@ class Status {
         float averageSpeed;
     } speeds;
     float averageSpeed;
+    float averageBattV;
```

Comment:
suggest that we group the two new variables in a single `batteryVoltage` struct - maybe `.average` and `isLow`?

Reply:
I guess if we're doing that, we'd add in the current (latest) voltage too